cli/command/trust/sign.go

```diff
@@ -64,7 +64,7 @@ func signImage(cli command.Cli, imageName string) error {
 			}
 
 			fmt.Fprintf(cli.Out(), "Created signer: %s\n", authConfig.Username)
-			fmt.Fprintf(cli.Out(), "Finished initializing %q\n", notaryRepo.GetGUN().String())
+			fmt.Fprintf(cli.Out(), "Finished initializing signed repository for %s\n", imageName)
```

Comment:
minor nit/note: I think this change is already in master (https://github.com/riyazdf/cli/blob/master/cli/command/trust/sign.go#L67), the diff coverage should go up if you rebase, and will keep the diff's scope smaller.